src/main/mule/sources/anypoint/platform/apis/api-call-exchange.xml

```diff
@@ -9,6 +9,13 @@ http://www.mulesoft.org/schema/mule/validation http://www.mulesoft.org/schema/mu
 
 	<flow name="api-call-exchange-assets-flow" doc:id="cb45e4dc-7fd3-476d-aff3-7c0a30556088" maxConcurrency="${anypoint.platform.apis.exchange.maxConcurrency}">
 		<logger level="DEBUG" doc:name="Logger" doc:id="c04d063d-9f26-41c8-9f2c-efd64bc5a457" message="Calling Exchange - Assets" />
+		<set-variable value="#[0]" doc:name="Set tries" doc:id="c7f338f3-ae84-4ab6-a3c1-164760fc3954" variableName="tries"/>
+		<set-variable value="#[p('anypoint.platform.apis.exchange.graphql.limit')]" doc:name="Set limit" doc:id="d8776b0e-9641-46a9-9aba-4c536fcdf572" variableName="limit"/>
```

Comment:
Hi Richard, I understand, it's because I use it later in three places:
- To calculate the offset on every iteration
- The GraphQL Query
- The condition to stop iterating

So I didn't want to have three references to the same property.